app/uk/gov/hmrc/eoricommoncomponent/frontend/services/countries/Countries.scala

```diff
@@ -67,19 +68,19 @@ object Countries {
   val islands: List[Country] =
     countries filter (c => mdgCountryCodes("/mdg-country-codes-islands.csv") contains c.countryCode)
 
+  private val iomCountry = (List(Country.apply("Isle of Man", "IM")), IsleOfManCountryPicker)
```

Comment:
Doesn't need to be private or all the other country lists could be private too. You don't have to explicitly call the apply method like this.